### PR TITLE
edit user form and implemented testing

### PR DESCRIPTION
**Pull request by justgarcia5**

### CODE REVIEW

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<div class='wrapper'>
+  <h2>Edit User </h2>
+<div class='card full-width'>
+  <div class='w-form'>
+  <%= form_for @user do |f| %>
```
Comment (jasononaquest) [inline app/views/users/edit.html.erb]: We should make use of the form partial here.

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -4,17 +4,18 @@
...
-        <p class="init-team-leader">Team Member</p>
+    <% @users.each do |user| %>
+      <%= link_to edit_user_path(user), :style=> "text-decoration: none;" do %>
+      <div class="card">
+        <h3 class="init-fullname"><%= user.fullname %></h3>
+        <p class="init-email"><%= user.email %></p>
+        <% if user.is_admin == true %>
+          <p class="init-is-admin">Admin</p>
+         <% end %>
+        <% if user.team_leader_id != nil and user.is_admin != true %>
```
Comment (jasononaquest) [inline app/views/users/index.html.erb]: I think these checks can be rewritten to not use `==` or `!=`. That is a generally more idiomatic approach when possible.

**Review by jasononaquest — CHANGES_REQUESTED** [review]

File: app/views/users/edit.html.erb (hunk 1 above)
Comment (kevinyu05) [reply app/views/users/edit.html.erb]: form partial has been added

File: app/views/users/index.html.erb (hunk 2 above)
Comment (kevinyu05) [reply app/views/users/index.html.erb]: Switched checks to be rewritten without == or !=

File: app/views/users/_users.html.erb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+
+<div class="init-list">
+  <% users.each do |user| %>
+    <%= link_to edit_user_path(user), :style=> "text-decoration: none;" do %>
+      <div class="card">
+        <h3 class="init-fullname"><%= user.fullname %></h3>
+          <p class="init-email"><%= user.email %></p>
+            <% if user.is_admin == true %>
```
Comment (jasononaquest) [inline app/views/users/_users.html.erb]: This section of logic could use the same treatment as we have in index. I'm sorry for not noticing it sooner.

**Review by jasononaquest — APPROVED** [review]

**Review by jasononaquest — CHANGES_REQUESTED** [review]
Fix the users partial to use similar logic to index and I think we'll be good to go.

File: app/views/users/_users.html.erb (hunk 3 above)
Comment (kevinyu05) [reply app/views/users/_users.html.erb]: Altered logic to represent same treatment in index. No worries!

**Review by jasononaquest — APPROVED** [review]
LGTM

**Pull request merged**